pykube/config.py

```diff
@@ -12,6 +12,15 @@
 from pykube import exceptions
 
 
+def _join_host_port(host, port):
+    """Adapted golang's net.JoinHostPort"""
+    template = "%s:%s"
```

Comment:
please use new style "format" instead of "%"

Reply:
Fixed